### PR TITLE
Feature/show initial board UI

### LINKED ISSUE #84: Show the InitialContent on firts Render
image

### PR DESCRIPTION
**Pull request by tomivm**
close #84

### CODE REVIEW

File: src/stores/board-slice.ts [hunk 1]
```diff
@@ -49,4 +55,9 @@ export const createBoardSlice: StateCreator<
       type: 'Board/setErrorOnBoardGeneration',
     });
   },
+  hideInitialBoard: () => {
```
Comment (RodriSanchez1) [inline src/stores/board-slice.ts:58]: You can extend this action functionability.

```typescript
 setShowInitialBoard: (isShowInitialBoard: boolean) => {
    set(() => ({ showInitialBoard: isShowInitialBoard }), false, {
      type: 'Board/HideInitialBoard',
    });
  },
```

File: src/stores/board-slice.ts [hunk 2]
```diff
@@ -15,17 +16,22 @@ export type BoardActions = {
   setBoard: (board: BoardRecord) => void;
   cleanBoard: () => void;
   setErrorOnBoardGeneration: () => void;
+  hideInitialBoard: () => void;
 };
 export type BoardSlice = BoardStoreRecord & BoardActions;
 
 export const defaultBoardState: {
+  showInitialBoard: boolean;
```
Comment (RodriSanchez1) [inline src/stores/board-slice.ts:24]: Change name to `showInitialContent`

File: src/stores/board-slice.ts [hunk 3]
```diff
@@ -15,17 +16,22 @@ export type BoardActions = {
...
   board: null;
   errorOnBoardGeneration: boolean;
 } = {
+  showInitialBoard: true,
   board: null,
   errorOnBoardGeneration: false,
 };
 
+const CLEAN_BOARD_STATE = { ...defaultBoardState, showInitialBoard: false };
```
Comment (RodriSanchez1) [inline src/stores/board-slice.ts:33]: I think if you clean the board, the state of  `showInitialContent` needs to be true.
Try to implement the loading  board view changin a little bit the logic

Comment (tomivm) [reply src/stores/board-slice.ts:33]: Please create a new issue for this.

File: src/stores/board-slice.ts (hunk 1 above)
Comment (tomivm) [reply src/stores/board-slice.ts:58]: sound good! it's not necessary yet

**Pull request merged**